rules/config

```diff
@@ -31,3 +31,6 @@ USERNAME = admin
 
 # PASSWORD - password for installer build
 PASSWORD = YourPaSsWoRd
+
+# SONIC_CONFIG_DEBUG - install debug packages
+SONIC_CONFIG_DEBUG = n
```

Comment:
can you follow other comments?

 # Uncomment next line to enable:
 # SONIC_CONFIG_DEBUG = y